sycl/test-e2e/lit.cfg.py

```diff
@@ -173,6 +173,33 @@
 if lit_config.params.get("igc-dev", False):
     config.available_features.add("igc-dev")
 
+# Map between architecture types and device name.
+device_name_arch_map = {
+    # <Device name> : Set of architectures types (and aliases)
+    # DG1
+    "gpu-intel-dg1": {"intel_gpu_dg1"},
```

Comment:
Do we really need a family which consists of a single device? I would suggest to update tests to use the exact target instead.

Reply:
True. I've renamed `dg1` tests in https://github.com/intel/llvm/pull/16432